perception/occupancy_map_monitor/src/occupancy_map_monitor.cpp

```diff
@@ -303,6 +318,7 @@ bool OccupancyMapMonitor::loadMapCallback(moveit_msgs::LoadMap::Request& request
   tree_->lockWrite();
   try
   {
+    tree_->enableChangeDetection(false);
```

Comment:
Setting change detection to false is necessary to make this pull request properly interact with pull request #602.